src/libcrun/cgroup.c

```diff
@@ -1839,7 +1839,7 @@ must_stop_proc (runtime_spec_schema_config_linux_resources *resources)
 int
 libcrun_cgroup_enter (struct libcrun_cgroup_args *args, libcrun_error_t *err)
 {
-  pid_t sigcont_cleanup __attribute__ ((cleanup (cleanup_sig_contp))) = -1;
+  __attribute__ ((unused)) pid_t sigcont_cleanup __attribute__ ((cleanup (cleanup_sig_contp))) = -1;
```

Comment:
the changes in this file seem to belong to a different commit.

What compiler gives the warning?

Reply:
@giuseppe I think its just the new version of `clang-check` on `f35`. `clang-check` tests fail with.
```console
src/libcrun/container.c:3392:69: error: variable 'process_cleanup' set but not used [-Werror,-Wunused-but-set-variable]
  cleanup_process_schema runtime_spec_schema_config_schema_process *process_cleanup = NULL;
                                                                    ^
```

Sure I'll add these checks in a different commit.

Reply:
Specifically this check https://github.com/containers/crun/tree/main/tests/clang-check but i agree this should be a separate commit.